README.md

```diff
@@ -1,8 +1,39 @@
-# game_of_life https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life
-Game Of Life
+# Game Of Life 
+[Convey's game of life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) is a zero-player game, meaning that its evolution is determined by its initial state, requiring no further input. One interacts with the Game of Life by creating an initial configuration and observing how it evolves, or, for advanced players, by creating patterns with particular properties.
+
 This an OOP implementation of Conway's game of life in C++. 
 Use of abstract classes and interface allows possibility of extension (inheriting from Board.h or NormalBoard.h) like:
- - Langton’s Ant
- - Brian’s Brain
- - Day and Night
- - etc..
+ - [Langton’s Ant](https://en.wikipedia.org/wiki/Brian%27s_Brain)
+ - [Brian’s Brain](https://en.wikipedia.org/wiki/Brian%27s_Brain)
+ - [Day and Night](https://en.wikipedia.org/wiki/Day_and_Night_(cellular_automaton))
+ - [Pretty print](https://chrome.google.com/webstore/detail/prettyprint/nipdlgebaanapcphbcidpmmmkcecpkhg?hl=en) 
+ - etc...
+ 
+To build current code use [Eclipse IDE for C/C++ Developers](https://www.eclipse.org/downloads/packages/release/helios/sr2/eclipse-ide-cc-developers).
+The instructions in Eclipse are:
+
+**(please delete .cproject and .project from your branch before importing it to eclipse. If possible add them to .gitignore)**
+1. File -> Import... 
+
+![file menu](https://github.com/Tawishi/game_of_life/blob/Issue%233/1.jpg)
+
+   -> Git -> Projects from Git 
+
+![git](https://github.com/Tawishi/game_of_life/blob/Issue%233/2.jpg)
+
+   -> Clone URI.
+
+![git_clone](https://github.com/Tawishi/game_of_life/blob/Issue%233/3.jpg)
+
+2. Set URI GIT_CLONE_URI.
+
+![Import_project](https://github.com/Tawishi/game_of_life/blob/Issue%233/issue2.jpg)
```

Comment:
set links to point to original repository

Reply:
What links? Please specify.